tests/data/intents/litexa/main.litexa

```diff
@@ -18,10 +18,55 @@ launch
     say "heard upper"
   when "lower case utterance"
     say "heard lower"
+  when OtherIntentName
+    or "other intent name"
+    -> someState
+
+someState
+  say "in some state"
+
```

Comment:
Would remove, unless there's a reason to have it in this test.

Reply:
Just additional verification that a transition to the top of this state occurred, and that the state's when handlers do not execute this part.